### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by chopperg**
managing-contribution-graphs-on-your-profile

### Why:

Closes [issue link]

### Check off the following:

- [x] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [x] This pull request impacts the contribution experience
  - [x] I have added the 'writer impact' label
  - [x] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

File: content/account-and-profile/setting-up-and-managing-your-github-profile/index.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
----
+docs/content/account-and-profile/setting-up-and-managing-your-github-profile/index.md---
```
Comment (chopperg) [inline content/account-and-profile/setting-up-and-managing-your-github-profile/index.md:1]: Sharp Development 2016 (c) Gregory Sharp

**Review by chopperg — COMMENTED** [review]
Notice

**Review by Pojielp — COMMENTED** [review]
0c4c99789f047a66c59cd8598f530756ab462cac

File: content/account-and-profile/setting-up-and-managing-your-github-profile/index.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
----
```
Comment (chopperg) [inline content/account-and-profile/setting-up-and-managing-your-github-profile/index.md:1]: -the-codeql-workflow.md-the-codeql-workflow.mdcontent/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/troubleshooting-the-codeql-workflow.mdhttps://github.com/testdouble/standard$ bundle exec standardrb
standard: Use Ruby Standard Style (https://github.com/testdouble/standard)
standard: Run `standardrb --fix` to automatically fix some problems.
  /Users/code/cli.rb:31:23: Style/Semicolon: Do not use semicolons to terminate expressions.

**Review by chopperg — COMMENTED** [review]
code-review/README.md

**Pull request closed without merging**